client-cli/build.gradle.kts

```diff
@@ -3,13 +3,17 @@ plugins {
     id("application")
     id("com.github.johnrengelman.shadow") version "7.0.0"
     `maven-publish`
+    `java-test-fixtures`
```

Comment:
nit: maybe not needed as we do not have test-fixtures in client-cli module.

Reply:
👍 I forgot to remove it